src/DynamoCoreWpf/ViewModels/Core/StateMachine.cs

```diff
@@ -802,16 +802,6 @@ internal bool HandlePortClicked(PortViewModel portViewModel)
 
                 if (this.currentState != State.Connection) // Not in a connection attempt...
                 {
-                    if (Keyboard.Modifiers == ModifierKeys.Alt &&
-                        portViewModel.NodeAutoCompleteCommand.CanExecute(portViewModel))
-                    {
-                        portViewModel.NodeAutoCompleteCommand.Execute(portViewModel);
-                        this.currentState = State.Connection;
-                        owningWorkspace.CurrentCursor = CursorLibrary.GetCursor(CursorSet.ArcSelect);
-                        owningWorkspace.IsCursorForced = false;
-                        return true;
-                    }
```

Comment:
Deleting the old code handling Alt+ mouse left click to trigger node Autocomplete